### PR TITLE
Check if mpv is installed

### PR DESCRIPTION
**Pull request by Kerumen**

### CODE REVIEW

**Comment by balthazar** [comment]
Nice, but doesn't the main purpose of samus, a mpv remote selector, makes this obvious?

**Comment by Kerumen** [comment]
True. I just wanted to avoid an ugly error and exit properly.

**Comment by meriadec** [comment]
Thx

File: src/samus.js [hunk 1]
```diff
@@ -55,6 +55,13 @@ class Samus {
     console.log('\n>>>>>>>> Launching mpv...\n')
 
     const child = spawn('mpv', ['--quiet', name])
+    child.on('error', err => {
+      if (err.code === 'ENOENT') {
+        console.log('\nPlease install mpv to use samus (https://mpv.io/).\n')
+        process.exit()
+      }
+    });
```
Comment (meriadec) [inline src/samus.js:63]: semicolon spotted

Comment (Kerumen) [inline src/samus.js:63]: ups sorry

**Pull request merged**